CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Added
 - Logo
+### Changed
+- `quote` decorator updated
```

Comment:
`quote` decorator bug fixed

Reply:
Fixed in [b6a4cb8](https://github.com/openscilab/nava/pull/15/commits/b6a4cb83e82cb19022a8cedfaebfdf64627be37c).